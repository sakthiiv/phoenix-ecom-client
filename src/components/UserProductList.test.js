import React from "react";
import { shallow } from "enzyme";
import UserProductList from "./UserProductList";

describe("userProductList", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<UserProductList />);
    expect(wrapper).toMatchSnapshot();
  });
});
