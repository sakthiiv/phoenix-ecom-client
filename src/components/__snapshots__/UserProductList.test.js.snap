// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`userProductList renders correctly 1`] = `
<div>
  <div
    className="list-container"
    key="0"
  >
    <div />
  </div>
  <div
    className="list-container"
    key="1"
  >
    <Row
      tag="div"
    >
      <div
        className="columns"
        key="1"
      >
        <ProductItem
          key="1111"
          productItem={
            Object {
              "category": "Electronics",
              "description": "Tv Description goes here.",
              "id": "1111",
              "image": "",
              "name": "TV",
              "price": "18,000",
            }
          }
        />
        <ProductItem
          key="2222"
          productItem={
            Object {
              "category": "Electronics",
              "description": "Mobile descriptions goes here.",
              "id": "2222",
              "image": "",
              "name": "Mobile",
              "price": "10,000",
            }
          }
        />
      </div>
    </Row>
  </div>
  <div
    className="list-container"
    key="2"
  >
    <div />
  </div>
</div>
`;
